src/gui/generalsettings.cpp

```diff
@@ -517,15 +519,15 @@ void GeneralSettings::slotStyleChanged()
 void GeneralSettings::customizeStyle()
 {
     // setup about section
-    QString about = Theme::instance()->about();
-    Theme::replaceLinkColorStringBackgroundAware(about);
-    _ui->aboutLabel->setText(about);
+    auto aboutText = Theme::instance()->about();
```

Comment:
```suggestion
    const auto aboutText = Theme::instance()->about();
```

Reply:
@mgallien Impossible, see` void Theme::replaceLinkColorStringBackgroundAware(QString &linkString);` that expects non-const reference